tests/checkpoint/helpers.py

```diff
@@ -8,7 +8,7 @@
 from packaging import version
 from torch.distributed.fsdp import FullyShardedDataParallel as FSDP
 from torch.distributed.fsdp.api import CPUOffload
-from torch.optim import adam
+from torch.optim import Adam
```

Comment:
is this backwards compatible? eg does this import work on torch 2.3?

Reply:
Yes

Reply:
(otherwise it wouldn't have passed 2.3 tests also)